sql/insert.go

```diff
@@ -19,15 +19,47 @@ package sql
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/cockroachdb/cockroach/client"
 	"github.com/cockroachdb/cockroach/proto"
 	"github.com/cockroachdb/cockroach/sql/parser"
 	"github.com/cockroachdb/cockroach/sql/privilege"
-	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+// TOOD(tamird): instead of tracking every write, use the failed CPut's
```

Comment:
fixed

On Tue, Sep 1, 2015 at 5:48 AM, Tobias Schottdorf <notifications@github.com>
wrote:

> In sql/insert.go
> <https://github.com/cockroachdb/cockroach/pull/2303#discussion_r38402188>:
>
> >  	"github.com/cockroachdb/cockroach/util/log"
> >  )
> >
> > +// TOOD(tamird): instead of tracking every write, use the failed CPut's
>
> s/TOOD/TODO/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2303/files#r38402188>.
>